public/JSONtoHTML.coffee

```diff
@@ -0,0 +1,211 @@
+# Variables
+xmlHttp = new XMLHttpRequest
+jsonFile = 'stdLib_base_test.json'
+
+getMainData = (data) ->
+  output = ''
+  modules = data.modules
+
+  i = 0
```

Comment:
even in such examples don't use meaningless names. As I understand `i` is just `moduleNumber` ?

Reply:
current iteration

Reply:
current iteration is not semantic meaning. If I tell you - this variable is "current iteration" you don't understand what exactly it is for. What this iteration is used for. Always use semantic names.